### DISCUSSION TITLE
Quarkus Zeebe Runtime v0.8.0 nearest update to use Zeebe Client Java v8.2.2

### DISCUSSION THREAD

**Discussion by nickhais** [quarkusio/quarkus#32779] [Q&A]
Hello!

I work with Quarkus and Camunda and we are on our way to migrate from Camunda **8.0** to **8.2**.

In our project we use [quarkus-zeebe](https://mvnrepository.com/artifact/io.quarkiverse.zeebe/quarkus-zeebe/0.8.0) version **0.8.0**. This dependence use inside [Zeebe Client Java](https://mvnrepository.com/artifact/io.camunda/zeebe-client-java) version **8.1.9**. But **8.1.9** is not support all our required features from Camunda 8.2 and the latest version of Zeebe Client Java is **8.2.2**.

Does Quarkus team plan to update [Zeebe Client Java](https://mvnrepository.com/artifact/io.camunda/zeebe-client-java) in [quarkus-zeebe](https://mvnrepository.com/artifact/io.quarkiverse.zeebe/quarkus-zeebe/0.8.0) to the LTS version in nearest feature?

**Comment by hausi78** [comment]
Does [quarkus-zeebe 0.9.0](https://mvnrepository.com/artifact/io.quarkiverse.zeebe/quarkus-zeebe/0.9.0) work for you?

**Comment by nickhais** [comment]
Hello!We are checking it right now.-----"Christian Hauser" ***@***.***> wrote: -----To: "quarkusio/quarkus" ***@***.***>From: "Christian Hauser" ***@***.***>Date: 04/23/2023 01:06AMCc: "Nick" ***@***.***>, "Author" ***@***.***>Subject: Re: [quarkusio/quarkus] Quarkus Zeebe Runtime v0.8.0 nearest update to use Zeebe Client Java v8.2.2 (Discussion #32779)
Does https://mvnrepository.com/artifact/io.quarkiverse.zeebe/quarkus-zeebe/0.9.0 work for you?
